cmd/metal-api/main.go

```diff
@@ -757,7 +765,7 @@ func initRestServices(audit auditing.Auditing, withauth bool, ipmiSuperUser meta
 		releaseVersion = pointer.Pointer(viper.GetString("release-version"))
 	}
 
-	restful.DefaultContainer.Add(service.NewAudit(logger.WithGroup("audit-service"), audit))
+	restful.DefaultContainer.Add(service.NewAudit(logger.WithGroup("audit-service"), pointer.FirstOrZero(audit)))
```

Comment:
Is there there some documentation about this behavior of only using the first auditing backend as source for the search?

Reply:
I added another flag `--auditing-search-backend` to let the user decide a specific backend.